python/coverage/action.yaml

```diff
@@ -0,0 +1,39 @@
+name: 'Upload code coverage'
+description: 'Upload code coverage report for pytests.'
+inputs:
+  python-version:
+    required: false
+    description: Version of python
+    default: "3.10.x"
+  code-coverage-path:
+    required: false
+    description: Path to pytests code coverage report dir.
+    default: "artifacts/html-coverage/"
+  aws-access-key-id: 
+    required: true
+    description: AWS Access Key ID
+  aws-secret-access-key:
+    required: true
+    description: AWS Secret Access Key
+  aws-region:
+    required: true
+    description: AWS Region
+    default: eu-central-1
+  aws-bucket:
+    required: true
+    description: AWS Bucket
+    default: github-code-coverage
+runs:
+  using: "composite"
+  steps:
+    - uses: unfor19/install-aws-cli-action@v1
```

Comment:
I think it should be https://github.com/PiwikPRO/actions/blob/da29c8af59d9103a2deb5e376828f6ca693b8341/coverage/action.yml#L23